lib/utils/docker.coffee

```diff
@@ -196,13 +196,23 @@ exports.runBuild = (params, options, getBundleInfo, logger) ->
 	doodles = require('resin-doodles')
 	transpose = require('docker-qemu-transpose')
 	path = require('path')
+	os = require('os')
 
 	# The default build context is the current directory
 	params.source ?= '.'
 	logs = ''
 	# Only used in emulated builds
 	qemuPath = ''
 
+	if os.platform() == 'win32'
+		logger.logWarn '''
+			SECURITY WARNING: You are building a Docker image from
+			Windows against a non-Windows Docker host.
+			All files and directories added to build context will have
+			'-rwxr-xr-x' permissions. It is recommended to double check
+			and reset permissions for sensitive files and directories.
+		'''
```

Comment:
Can you add a comment here explaining that this is imitating the Docker client behaviour too, and link to the corresponding code for that?